src/Language/es/Auth.php

```diff
@@ -0,0 +1,89 @@
+<?php
```

Comment:
Please fix the file encoding. We only accept UTF-8 file.

```consolse
$ file src/Language/es/Auth.php 
src/Language/es/Auth.php: PHP script text, ISO-8859 text
```

Reply:
@treborin [here ](https://www.kindacode.com/article/how-to-change-file-encoding-in-vs-code/)is a good guide.

Let me know if you need more help

Reply:
![image](https://user-images.githubusercontent.com/31905622/184476833-b11fb261-1c0a-4529-b2cc-1832a0f5766b.png)

Reply:
![image](https://user-images.githubusercontent.com/31905622/184477052-02a76adc-e284-42c3-b04e-b26bd551dd5d.png)